docusaurus/docs-internals/editor-controls/multivalue-controls/gallery.md

```diff
@@ -162,43 +251,80 @@ In this example, the user cannot remove the last item in the gallery.
 }
 ```
 
-### Return value
+#### Default value examples
 
-Returns a stringified array of objects with the following values:
+In this example, setting a default value for the gallery control.
 
 ```js
 {
-  id: number;
-  uid: string;
-  fileName: string;
-  width: number;
-  height: number;
+  id: "slides",
+  type: "gallery",
+  default: {
+    value: '[
+      {
+        id: "1",
+        uid:"dbe6479df22c4ef2ac6553aa241d2c5c.jpg",
+        fileName: "image1.jpg",
+        width: 800,
+        height: 600
+      },
+      {
+        id: "2",
+        uid:"fbe6479df22c4ef2ac6555te241d2c5c.jpg",
+        fileName: "image2.jpg",
+        width: 1024,
+        height: 768
+      }
+    ]'
+  }
 }
 ```
-Example:
-```js
-{
-  value: '[
-    {
-      "id": 1,
-      "uid": "dbe3a79df22c4ef2ac6553aa241d2c5c.jpg",
-      "fileName": "image1.jpg",
-      "width": 1600,
-      "height": 1042
-    },
-    {
-      "id": 2,
-      "uid": "a7c45ea83554c81024b84d724fd8286d.jpg",
-      "fileName": "image2.jpg",
-      "width": 640,
-      "height": 359
-    }
-  ]'
+
+#### Usage in HTML example
+
+```tsx
+import { Brizy } from "@brizy/core";
+import { Gallery } from "./Gallery";
+import React, { JSX } from "react";
+
+interface Props {
+  fileName: string;
 }
-```
 
-`id` - The index of the image in the gallery; <br/>
-`uid` - The unique identifier of the image; <br/>
-`fileName` - The unique identifier of the image; <br/>
-`width` - The width of the image; <br/>
-`height` - The height of the image; <br/>
+const Gallery = (props: Props): JSX.Element => {
+  const { fileName } = props;
+
+  return (fileName === "" ? <p>Add a valid image</p> : <Gallery />)
+};
```

Comment:
not clear example

how i will do: i will parse the images from gallery control, then i will get an array and i will check check:
```ts
array.length > 0 ? <SomeComponent /> : <AnotherComponent />
```